command/commands.go

```diff
@@ -8,12 +8,13 @@ import (
 	"os/signal"
 	"syscall"
 
+	"github.com/mitchellh/cli"
```

Comment:
Could you revert this change please?

Reply:
This was automatic through `make fmt`. Should I revert it anyways?

Reply:
It doesn't for me.  Are you sure that's the source of the change?